projects/goharbor/harbor/Makefile

```diff
@@ -0,0 +1,41 @@
+BASE_DIRECTORY=$(shell git rev-parse --show-toplevel)
+GIT_TAG?=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+REPO=harbor
+REPO_OWNER=goharbor
+REPO_SUBPATH=./src
+
+BASE_IMAGE_NAME?=eks-distro-base
```

Comment:
I think probably because of that entrypoint we are using the standard base here which has bash, but Id really prefer not to... Anyway we could avoid the entrypoint?  Or maybe we need to make a minimal image with bash that we use only in specific places like this.

Reply:
Harbor uses scripts heavily for its service initialization and configuration. I don't think it is possible to implement those complicated logics (conditional & loops) with vanilla commands in Dockerfile. Yes, bash is the major reason I go for the standard image. Besides that, a quite few packages are also needed in the runtime - tzdata, shadow-utils, cronie, rsyslog, logrotate, tar, gzip, sudo, etc.